lib/travis/api/build/app.rb

```diff
@@ -39,6 +40,10 @@ def api_tokens
             @api_tokens ||=
               Travis::Build.config.api_token.to_s.split(',').map(&:strip)
           end
+
+          def librato_disabled? 
```

Comment:
ack, it is not. Left over from trying to track down some config issues. So I broke out the above logic, but rolled it back but not completely.